spec/src/ast/core/typing/bidirectional.makam

```diff
@@ -109,7 +113,7 @@
     synth_type Context (function_type InputType OutputType) type_type :-
         is_type Context InputType,
         context.eval Context InputType InputType',
-        context.add_param Context InputType' Context'.
+        context.add_param Context InputType' Context',
         is_type Context' OutputType.
```

Comment:
This was unfortunate! 🤦 🤦

Reply:
@astampoulis Not sure if Makam could give some warnings on unused variables here? This meant that a bunch of my tests were passing when they should have been failing!

On a related note I've also run into confusing type errors when `:-` is left off by accident, but I think that's pretty hard to avoid, given the λProlog syntax. Thankfully it's less critical because you actually get an error in that case (although it's scary from a user perspective).